README.md

```diff
@@ -55,13 +57,36 @@ Any merged PRs with an **Editor Review** or **Reviews Complete** label will be p
 
 ## Building and Validating
 
-If you've downloaded the repo and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review them before you commit changes or create pull requests. 
-
-1. Download the Mattermost docs repo onto a machine with Python installed.
-2. Install [pipenv](https://docs.pipenv.org/): `pip install pipenv`. 
-3. `cd` into the cloned repository.
-4. Install the required packages: `pipenv install`.
-5. Build the doc set using `make html`. This generates files in `/build` directory.
-6. Navigate to the `/build` directory to preview the page/s you've edited. 
+If you've downloaded the repository and are editing Mattermost documentation on your local machine, you can generate the HTML files from markdown in the `/source` directory. You can review your changes before you commit them or create pull requests.
+
+**Note:** Commands can be executed on Linux, Mac, and Windows (using Powershell).
+
+1. Open a terminal window, then clone a forked copy of the documentation repository by running the following command::
+```sh
```

Comment:
@justinegeffen - do we need preceding spaces in order for the code-formatted examples to display correctly? If so, will impact the following steps below as well.

Reply:
I'll confirm.

Reply:
No, it looks like it displays as expected without them.